shared/templates/static/ansible/auditd_data_retention_flush.yml

```diff
@@ -0,0 +1,13 @@
+# platform = multi_platform_rhel,multi_platform_fedora
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: Configure auditd Flush Priority
+  lineinfile:
+    dest: /etc/audit/auditd.conf
+    regexp: '.*flush.*'
+    line: flush = data
+  notify: reload auditd
```

Comment:
let's comment it or something so that we can grep for it when we implement handlers